includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -172,17 +180,23 @@ public function stripe_check_environment() {
 					$this->add_admin_notice( 'curl', 'notice notice-warning', __( 'WooCommerce Stripe - cURL is not installed.', 'woocommerce-gateway-stripe' ), true );
 				}
 			}
-			// @codeCoverageIgnoreEnd
 
+			// @codeCoverageIgnoreEnd
 			if ( empty( $show_keys_notice ) ) {
 				$secret = WC_Stripe_API::get_secret_key();
 				// phpcs:ignore
 				$should_show_notice_on_page = ! ( isset( $_GET['page'], $_GET['section'] ) && 'wc-settings' === $_GET['page'] && 0 === strpos( $_GET['section'], 'stripe' ) );
 
 				if ( empty( $secret ) && $should_show_notice_on_page ) {
 					$setting_link = $this->get_setting_link();
-					/* translators: 1) link */
-					$this->add_admin_notice( 'keys', 'notice notice-warning', sprintf( __( 'Stripe is almost ready. To get started, <a href="%s">set your Stripe account keys</a>.', 'woocommerce-gateway-stripe' ), $setting_link ), true );
+
+					$notice_message = sprintf(
+					/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
+						esc_html__( 'Stripe is almost ready. To get started, %1$sset your Stripe account keys%2$s.', 'woocommerce-gateway-stripe' ),
+						'<a href="' . $setting_link . '">',
```

Comment:
`$this->get_setting_link()`[ is a wrapper](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/includes/admin/class-wc-stripe-admin-notices.php#L369) of `admin_url()`, so we might want to escape those.

Other instances are
[1](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR212)
[2](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR227)
[3](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR243)

Reply:
fixed by 3250324ea2646d5d625d565791aa82f275012c7e